tests/test_aiidalab.py

```diff
@@ -130,12 +130,16 @@ def test_install_apps_from_stable(aiidalab_exec, package_name, nb_user, variant)
     assert "dependency conflict" not in output
     assert f"Installed '{package_name}' version" in output
 
+    # Uninstall the package to make sure the test is repeatable
+    aiidalab_exec(f"aiidalab uninstall --yes --force {package_name}", user=nb_user)
```

Comment:
I think we need to implement this as a separate fixture (`aiidalab install`?) and have this as a teardown, not as part of the test itself, otherwise things will still be flaky...